api/python/cell_census/src/cell_census/_open.py

```diff
@@ -103,27 +110,25 @@ def open_soma(
 
 
 def get_source_h5ad_uri(dataset_id: str, *, census_version: str = "latest") -> CensusLocator:
-    """
-    Open the named version of the census, and return the URI for the ``dataset_id``. This
-    does not guarantee that the H5AD exists or is accessible to the user. Raises an
-    error if ``dataset_id`` or ``census_version`` are unknown [lifecycle: experimental].
-
-    Parameters
-    ----------
-    dataset_id : ``str``
-        The ``dataset_id`` of interest
-    census_version : ``Optional[str]``
-        The census version
-
-    Returns
-    -------
-    ``CensusLocator`` : the URI and optional S3 region for the source H5AD
-
-    Examples
-    --------
-    >>> cell_census.get_source_h5ad_uri("cb5efdb0-f91c-4cbd-9ad4-9d4fa41c572d")
-    {'uri': 's3://cellxgene-data-public/cell-census/2022-12-01/h5ads/cb5efdb0-f91c-4cbd-9ad4-9d4fa41c572d.h5ad',
-    's3_region': 'us-west-2'}
+    """Open the named version of the census, and return the URI for the ``dataset_id``. This
+    does not guarantee that the H5AD exists or is accessible to the user.
+
+    [lifecycle: experimental]
+
+    Args:
+        dataset_id: The ``dataset_id`` of interest.
+        census_version: The census version.
+
+    Returns:
+        A :class:`CensusLocator` object that contains the URI and optional S3 region for the source H5AD.
```

Comment:
I noticed some of these have the tag `:py:class:` and some have `:class:` - are both correct?

Reply:
They're both correct - I used `:py:class:` for external docs since that's the prefix exported by Sphinx. But since both work, I will settle on `:class:`.